packages/nextjs/components/scaffold-eth/Input/AddressInput.tsx

```diff
@@ -11,22 +11,32 @@ import { CommonInputProps, InputBase, isENS } from "~~/components/scaffold-eth";
 export const AddressInput = ({ value, name, placeholder, onChange, disabled }: CommonInputProps<Address | string>) => {
   // Debounce the input to keep clean RPC calls when resolving ENS names
   // If the input is an address, we don't need to debounce it
-  const _debouncedValue = useDebounce(value, 500);
+  const [_debouncedValue] = useDebounceValue(value, 900);
```

Comment:
Lol `900ms` came from just brute force trying and seems like a good spot

1. fast typer wont need to wait too much before it starts resolving adderss
2. Gives a a good decent amount of window for slow typers to full complete domain 

But lol it is kind of personal preference, so please feel free to suggest any better number 🙌

Reply:
voting for 500 since I think there are much less slow typers than fast typers between our users